src/pam/pam_oslogin_login.cc

```diff
@@ -104,7 +104,7 @@ pam_sm_acct_mgmt(pam_handle_t* pamh, int flags, int argc, const char** argv) {
 
   if (http_code != 200) {
     PAM_SYSLOG(pamh, LOG_INFO, "Failed to validate organization user %s has "
-                               "login permission, got HTTP response code %d.",
+                               "login permission, got HTTP response code %ld.",
```

Comment:
I personally, don't need it, a compiler need it.
If you check the code, you'll notice that I do not defined the type of `http_code`. It was defined long before me. I'm only fixing formatting, because a compiler complains in that point for referring integer while long is defined.